test/test_utils.py

```diff
@@ -317,5 +325,18 @@ def test_draw_keypoints_errors():
         utils.draw_keypoints(image=img, keypoints=invalid_keypoints)
 
 
+def test_random_colors():
```

Comment:
Some tests so that we are OK with the implementation. I guess these aren't very necessary but anyways good to have and don't hurt much.